plugins/custom_calyptia/calyptia.c

```diff
@@ -338,8 +339,14 @@ static int cb_calyptia_init(struct flb_custom_instance *ins,
     flb_input_set_property(ctx->i, "scrape_on_start", "true");
     flb_input_set_property(ctx->i, "scrape_interval", "30");
 
+    if (ctx->fleet_name || ctx->fleet_id) {
+        is_fleet_mode = FLB_TRUE;
+    } else {
```

Comment:
add the else in a new line: https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md#braces-usage-on-conditionals-loops-and-functions

Reply:
I went ahead and added the required new lines for `if` and `else` in the `custom_calyptia` and `in_calyptia_fleet` plugins (in separate commits).